src/Engine/ProtoCore/Utils/StringUtils.cs

```diff
@@ -1,12 +1,16 @@
 using ProtoCore.DSASM;
 using System;
+using System.Collections.Generic;
 using System.Linq;
 using System.Text.RegularExpressions;
 
 namespace ProtoCore.Utils
 {
     public static class StringUtils
     {
+        internal const string DynamoPreferencesNumberFormat = nameof(DynamoPreferencesNumberFormat);
```

Comment:
where is DynamoPreferencesNumberFormat defined?

Reply:
you are looking at it.
When this is the format specifier, we just use the current `ProtoCore.Mirror.MirrorData.PrecisionFormat` which is set by the prefs.

I actually think this is a bad idea but I believe it was part of the AC to control the output using the display number format.

I think it 's a bad idea because it makes sharing graphs harder.

Reply:
Oh, I did not know you could define strings this way `symbol = nameof(symbol)`

Reply:
I'm fine with these as Experimental - Can we add in the details into an Extended Node Help @mjkkirschner ? 

Longer term agree this makes more sense with dropdowns and inputs that also work well with into ML Node Autocomplete.

Reply:
@Amoursol @Jingyi-Wen  - sure I can make sure these nodes have some .md files that explain how to use them.

As far as making sure users understand they are experimental - do you have any objections to naming them with `Experimental` in the name, or adding a new category for experimental nodes?

I think two big issues with the current experimental toggle we have are:
1. If I use the node from the library, I have no idea it's controlled by the experimental toggle.
2. Usually we make experimental features off by default so they have low discoverability where the whole point is to get as much feedback as possible.

Consider it an experiment to see how we can experiment better?